### PR TITLE
[WIP] Docker setup improvements

### PR DESCRIPTION
**Pull request by marksteve**
Got to make docker deploys work! Used this repo to test: https://github.com/marksteve/freight-test-web Should I create an extra page to discuss docker setups? There's still a manual step to create machines though.

- [x] Add worker service
- [x] Add docker orchestration tools inside service images
- [ ] Update docs

### CODE REVIEW

File: docker-entrypoint [hunk 1]
```diff
@@ -5,6 +5,8 @@ if [ ! -e venv/bin/activate ]; then
 fi
 source venv/bin/activate
 python setup.py develop
-npm install
-npm run postinstall
+if [ "$1" == "bin/web" ]; then
+  npm install
+  npm run postinstall
```
Comment (dcramer) [inline docker-entrypoint]: Doesn't this always run with npm install?

Comment (marksteve) [inline docker-entrypoint]: npm doesn't run install hooks when you run it as root so I had to explicitly run that script.

**Comment by dcramer** [comment]
Thanks!

**Comment by dcramer** [comment]
Regarding the docs, I think a separate page is fine

**Pull request merged**